flutter_local_notifications_windows/lib/src/details/notification_text.dart

```diff
@@ -0,0 +1,51 @@
+import "package:xml/xml.dart";
+
+import "notification_part.dart";
+
+/// Where text can be placed in a Windows notification.
+enum WindowsTextPlacement {
+  /// Shown at the bottom of the notification body in smaller text.
+  attribution,
+}
+
+/// Text in a Windows notification.
+///
+/// See: https://learn.microsoft.com/en-us/uwp/schemas/tiles/toastschema/element-text
+class WindowsNotificationText extends WindowsNotificationPart {
+  /// Creates text for a Windows notification.
+  const WindowsNotificationText({
+    required this.text,
+    this.centerIfCall = false,
+    this.isCaption = false,
+    this.placement,
+    this.languageCode,
+  });
+
+  /// The text being displayed.
+  final String text;
+
+  /// Whether to center this text. Only relevant if in an incoming call.
+  final bool centerIfCall;
+
+  /// Whether the text should be smaller like a caption.
+  final bool isCaption;
+
+  /// The placement of this text. Null indicates default.
```

Comment:
Can you move second sentence to another paragraph?

Reply:
Done